quantum/template/avr/config.h

```diff
@@ -113,7 +113,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
 /* key combination for magic key command */
-/* defined by default. To change, uncomment and set to the combination you want */
+/* defined by default; to change, uncomment and set to the combination you want */
```

Comment:
It bugs me that the first part isn't a sentence, but if I made it a sentence it would be inconsistent with all the other similar blocks. This way the whole thing can be passed off as one (pseudo)sentence.

Reply:
This better? 
```suggestion
/* This is defined by default. To change, uncomment and set to the combination you want */
```

Reply:
Yeah, though I still prefer the semicolon as the capitalized text doesn't match the other similar comments in these files. It's definitely not that important, though.

Let's stick with the semicolon for now.